paymentservice/README.md

```diff
@@ -0,0 +1,26 @@
+# Payment Service
+
+Payment Service is responsible for validating the card details and sending a mock payment id. The validation is done by checking length, performing Luhn algorithm validation, validating the card company, and checking the expiry date. This microservice shows some usage of low-level operations to implement the validation algorithm.
+
+
+```bal
+isolated function isLuhnValid() returns boolean|error {
```

Comment:
After the code improvements, we have missed to update the codes in the readme. These are the old implementations

Reply:
yes, I'll check these places and update the latest code.

Reply:
I have observed that we have avoided using `stubs:` in these examples. Is this international? or these source code added before we these stubs are implemented.

Reply:
Acually these were added before moving the stub files to a separate module.

Reply:
OK, then I'll update those code snippets in the readme files and send another PR.